lib/logstash/outputs/email.rb

```diff
@@ -131,7 +107,7 @@ def register
         delivery_method :@via, options
```

Comment:
not sure this works either, could you open an issue for this and I will make sure to track this down in a next PR? Thanks  @guyboertje